extensions/amp-position-observer/validator-amp-position-observer.protoascii

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: { # amp-position-observer script
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-position-observer"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-position-observer>
+  html_format: AMP
+  tag_name: "AMP-POSITION-OBSERVER"
+  requires_extension: "amp-position-observer"
+  attrs: {
+    name: "intersection-ratios"
+    # "0", "0.1" , "1", "0 1", "0 0.5", "0.5 1", "1 1", etc...
+    value_regex: "^([0]*?\.\d*$|1$|0$)|([0]*?\.\d*|1|0)\s{1}([0]*?\.\d*$|1$|0$)"
```

Comment:
Note that `value_regex` needs `\` to be escaped to work properly. For instance `\d` should be `\\d`. To handle this, we usually include the unescaped version above in a comment. For example see [amp-timeago](https://github.com/ampproject/amphtml/blob/master/extensions/amp-timeago/validator-amp-timeago.protoascii#L38).